vms/platformvm/state/state_test.go

```diff
@@ -500,7 +501,7 @@ func newStateFromDB(require *require.Assertions, db database.Database) State {
 		&config.Config{
 			Validators: vdrs,
 		},
-		&snow.Context{},
+		&snow.Context{Log: logging.NoLog{}},
```

Comment:
What is this for?

Reply:
Got some test failing, will check what exactly

Reply:
Then either revert this change, or chnage license header (??)